Fix integration tests for LocalTerra as much as possible

inon-man opened the pull request:
## Description
Fix broken integration tests for preparing next Terra Classic upgrade.

fragwuerdig reviewed (commented):
Why did you close the PR?

inon-man committed:
Fix integration tests for LocalTerra as much as possible

inon-man committed:
feat: support computeTax for Terra Classic
feat: add rebel-2 to Classic network
feat: use uluna as default gas price for Classic
